### DISCUSSION TITLE
AA and AAA Dispenser Issues

### DISCUSSION THREAD

**Discussion by Minijuice24** [aderusha/DDD-Printable-Wall-Control-System#88] [Q&A]
Hello!

Has anyone sucessfully created the AA and AAA dispensers? I'm new to this and unfortunately can't figure out the pieces necessary to make it work.

2x "2x2 AA Battery Dispenser Tower"
1x "2x2 AA Battery Dispenser Base Right"
1x "2x2 AA Battery Dispenser Base Left"

That much makes sense. Now, to glue them together, the closest I came to a working solution was:

2x "2x2 Locking Spacer"
1x "4x0 Flat Left"
1x "4x0 Flat Right"
8x "4x10x8mm Pin"

With all of this together, the final product fits on the wall, but neither AA or AAA batteries fit in their respective towers. They're each too long. What am I missing?

**Comment by Rwings39** [comment]
@Minijuice24 Were you ever able to figure this out?  I was just about to ask the same question.  I tried more or less the same thing for the AA, except I used:

2x "2x1 Locking Spacers"
1x "2x2 Angle Bracket Left"
1x "2x2 Angle Bracket Right"
8x "4x10x8mm Pin"

Mine was just slightly too narrow to fit the AA batteries.  It looks like the blocks may be spaced to have a Flat Left and Flat Right on the inside (two hooks per half with no spacer?).  I feel like the 2x1 Locking Spacer I used would have worked if the tracks for the batteries were just a little deeper.

Edit: After looking at the “Flat” Side Pieces you mentioned, I noticed there is a “4x0 Flat Center” piece as well that looks to be just a bit wider than the “2x1 Locking Spacer” I tried, and probably the correct size.  I’m going to print one shortly to see if that works.

**Comment by Rwings39** [comment]
For anyone reading this in the future, yes it did require the two Battery Dispenser Base Pieces (left and right), two Dispenser Towers (for added height), two 4x0 Flats (left and right), and a 4x0 Flat Center.  The flat center gives the correct distance between the two dispenser pieces for both the AA and AAA dispensers.
